Sources/CleevioUI/Views/Inputs/CleevioInputField.swift

```diff
@@ -101,41 +129,43 @@ public struct CleevioInputField<
     }
 
     var state: InputFieldState {
-        .init(
-            isFocused: isFocused || configuration.isExternallyFocused,
+        let isFocused = isFocused || configuration.isExternallyFocused
+        return .init(
+            isFocused: isFocused,
             isEnabled: isEnabled,
-            isError: error != nil
+            isError: configuration.errorPresentationCondition(isFocused, error)
         )
     }
 
+    var isErrorPresented: Bool {
+        state.isError
+    }
+
     public var body: some View {
         VStack(alignment: .leading, spacing: 6) {
             configuration.title(state)
 
-            content(state)
-                .focused($isFocused)
-                .padding(configuration.contentPadding)
-                .background(configuration.background(state))
-                .overlay { configuration.overlay(state) }
-                .contentShape(Rectangle())
-                .onTapGesture {
-                    isFocused = true
-                }
+            VStack(alignment: .leading, spacing: .zero) {
+                content(state)
+                    .focused($isFocused)
+                    .padding(configuration.contentPadding)
+                    .background(configuration.background(state))
+                    .overlay { configuration.overlay(state) }
+                    .contentShape(Rectangle())
+                    .onTapGesture {
+                        isFocused = true
+                    }
 
-            if let error {
-                configuration.errorLabel(error)
-                    .transition(
-                        .asymmetric(
-                            insertion: .move(edge: .leading),
-                            removal: .move(edge: .leading)
-                        )
-                    )
+                configuration.errorLabel(error ?? "")
```

Comment:
I understand what we try to achieve here, my gut tells me` errorLabel != nil` should be basic condition, as there is nothing to be shown but empty space which is not something we probably want, right?